test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -211,14 +211,15 @@ private HapiApiSpec whitelistingAliasedContract() {
 								DEFAULT_PAYER, WHITELISTER, literalInitcodeFor("Whitelister"))),
 						sourcing(() -> createLargeFile(
 								DEFAULT_PAYER, CREATOR, literalInitcodeFor("Creator"))),
+//						uploadInitCode(WHITELISTER, CREATOR),
```

Comment:
we should delete these lines if we don't need them anymore instead of commenting them out